spec/factories/bikes.rb

```diff
@@ -100,15 +100,15 @@
     end
 
     trait :blue_trek_930 do
+      frame_model { "930" }
```

Comment:
Any particular reason for this change?

Reply:
Mainly vestigial (added a before(:save), turned out not to need it) but the attributes are sorted now.

Reply:
Cool. Just was curious.